blocked-edges/4.18.0-MCORestoreCon.yaml

```diff
@@ -0,0 +1,9 @@
+to: 4.18.0
+from: 4[.]17[.].*
+fixedIn: 4.18.13
```

Comment:
This only needs to be defined in the most-recent exposed patch release, so if you keep it in the 4.18.12 file, you can drop it from all the others.

Reply:
done!